apps/web/src/pages/Pools/AssessmentPlanBuilderPage/AssessmentPlanBuilderPage.tsx

```diff
@@ -97,13 +108,19 @@ export const AssessmentPlanBuilder = ({ pool }: AssessmentPlanBuilderProps) => {
               data-h2-flex-direction="base(column) l-tablet(row)"
               data-h2-align-items="base(flex-start) l-tablet(center)"
             >
-              {/* TODO: switch to submit button */}
-              {intl.formatMessage({
-                defaultMessage: "Save plan and go back",
-                id: "Rbp02p",
-                description:
-                  "Text on a button to save the assessment plan and return to the pool page",
-              })}
+              <Link
+                mode="solid"
+                color="secondary"
+                href={routes.poolView(pool.id)}
+              >
+                {/* Doesn't actually save anything */}
+                {intl.formatMessage({
+                  defaultMessage: "Save plan and go back",
+                  id: "Rbp02p",
+                  description:
+                    "Text on a button to save the assessment plan and return to the pool page",
```

Comment:
Isn't save buttons that don't do anything just confusing for everyone involved?

Reply:
Particularly when you have a second button next to it, "cancel", that also does the same thing

Reply:
Yeah, it's weird but we do it on the application, too.  I left that comment to acknowledge that it's not a mistake.

Reply:
Confirmed with Jerbo - it's a "behavioral save".